dev/ese/src/ese/jetapi.cxx

```diff
@@ -1317,7 +1317,7 @@ VOID INST::TraceStationId( const TraceStationIdentificationReason tsidr )
 LONG INST::iInstanceActiveMin = lMax;
 LONG INST::iInstanceActiveMac = 1;
 LONG INST::cInstancesCounter = 0;
-
+    
```

Comment:
Spaces changed.

Reply:
Grr, I thought I fixed this file, and oslayer_txt.cxx